components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -203,6 +211,8 @@
   $: {
     internalProps.open_hours = open_hours;
   }
+
+  $: internalProps.custom_fields = customFields;
```

Comment:
For those coming across internalProps / buildInternal / etc. for the first time in awhile: some extra hoops are being jumped through to allow for us to both have a prop be read via manifest/passed-prop, and also let it be editable.